### PR TITLE
📝  Add docstring check to ruff and fix respective errors

### PR DESCRIPTION
**Pull request by nquetschlich**

### CODE REVIEW

File: src/mqt/bench/benchmarks/qiskit_application_nature/groundstate.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
+"""Groundstate benchmark definition."""
```
Comment (burgholzer) [inline src/mqt/bench/benchmarks/qiskit_application_nature/groundstate.py:1]: There is a comment right below this top level one. Consider combining the two.

File: src/mqt/bench/benchmarks/qiskit_application_finance/portfolioqaoa.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+"""Portfolio QAOA benchmark definition."""
```
Comment (burgholzer) [inline src/mqt/bench/benchmarks/qiskit_application_finance/portfolioqaoa.py:1]: There is a comment right below this top level one. Consider combining the two.

File: src/mqt/bench/benchmarks/qiskit_application_finance/pricingput.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+"""Pricing put benchmark definition."""
```
Comment (burgholzer) [inline src/mqt/bench/benchmarks/qiskit_application_finance/pricingput.py:1]: There is a comment right below this top level one. Consider combining the two.

File: src/mqt/bench/benchmarks/qiskit_application_optimization/routing.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+"""Routing benchmark definition."""
```
Comment (burgholzer) [inline src/mqt/bench/benchmarks/qiskit_application_optimization/routing.py:1]: There is a comment right below this top level one. Consider combining the two.

**Review by burgholzer — APPROVED** [review]
Nice! Many thanks! 🙏 
Just a couple of suggestions where i think it might make sense to combine comments.

File: src/mqt/bench/benchmarks/qiskit_application_nature/groundstate.py (hunk 1 above)
Comment (nquetschlich) [reply src/mqt/bench/benchmarks/qiskit_application_nature/groundstate.py]: Fixed it.

File: src/mqt/bench/benchmarks/qiskit_application_finance/portfoliovqe.py [hunk 5]
```diff
@@ -1,3 +1,4 @@
+"""Portfolio VQE benchmark definition."""
```
[the remark this answers is not in the document]
Comment (nquetschlich) [reply src/mqt/bench/benchmarks/qiskit_application_finance/portfoliovqe.py]: Fixed it.

File: src/mqt/bench/benchmarks/qiskit_application_finance/pricingput.py (hunk 3 above)
Comment (nquetschlich) [reply src/mqt/bench/benchmarks/qiskit_application_finance/pricingput.py]: Fixed it.

File: src/mqt/bench/benchmarks/qiskit_application_ml/qnn.py [hunk 6]
```diff
@@ -1,3 +1,5 @@
+"""Quantum Neural Network benchmark definition."""
```
[the remark this answers is not in the document]
Comment (nquetschlich) [reply src/mqt/bench/benchmarks/qiskit_application_ml/qnn.py]: Fixed it.

File: src/mqt/bench/benchmarks/qiskit_application_optimization/tsp.py [hunk 7]
```diff
@@ -1,3 +1,4 @@
+"""TSP benchmark definition."""
```
[the remark this answers is not in the document]
Comment (nquetschlich) [reply src/mqt/bench/benchmarks/qiskit_application_optimization/tsp.py]: Fixed it.

File: src/mqt/bench/benchmarks/qiskit_application_finance/portfolioqaoa.py (hunk 2 above)
Comment (nquetschlich) [reply src/mqt/bench/benchmarks/qiskit_application_finance/portfolioqaoa.py]: Fixed it.

**Pull request merged**